api/middleware/roleAuthentication.js

```diff
@@ -0,0 +1,21 @@
+const roleAuthentication = (...args) => (req, res, next) => {
+  // role type is inside req.profile.body
+  //const { type } = req.profile
```

Comment:
may want to remove duplicate line

Reply:
yeah agree with Ashley you don't want to have those commented out code
I would say drop this pr fix the issue and send another PR